javascript/builtins/Promise.json

```
@@ -605,7 +612,7 @@
               "webview_ios": "mirror"
             },
             "status": {
-              "experimental": true,
+              "experimental": false,
```

@queengooborg I don't think https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#setting-experimental specifies what to do if one browser supports the feature, and another browser supports it only behind a flag. Should this remain experimental?

The first example may suggest that it should remain experimental for now:

> An API supported in Chrome and Firefox, without flags or prefixes.

```suggestion
              "experimental": true,
```

---

I think it should remain experimental. I'll commit this.